### ISSUE TITLE
Branches with slashes in the name missing from switcher

### ISSUE THREAD

**Issue by r24y** [atom/github#857]
### Prerequisites

- [X] Have you tried to reproduce the problem in [Safe Mode][safe-mode]?
- [X] Have you followed all applicable steps in the [debugging guide][debugging-guide]?
- [X] Have you checked the FAQs on the [message board][message-board] for common solutions?
- [X] Have you checked that your issue [isn't already filed][already-filed]?

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

Branches containing a slash (`/`) in the name do not show up in the branch list.

### Steps to Reproduce

1. In Atom, open a project that has a Git repository.
2. From a terminal, run `git checkout -b bug/branches/with-slashes`.

**Expected behavior:**

Newly created branch should show up in list.

**Actual behavior:**

Branch absent from list; only branches without slashes (such as `master`, `develop`) are displayed.

**Reproduces how often:**

Every time.

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

- Atom Beta 1.18.0-beta0
- APM:
```
apm  1.17.0
npm  3.10.5
node 6.9.5 x64
python 2.7.13
git 2.11.0
```
- OS X 10.12.4

### Additional Information

I looked into the code, and it looks like the branch list is coming from [here](https://github.com/atom/github/blob/master/lib/git-shell-out-strategy.js#L593). Running that command in the terminal correctly produced the full list of branches.

**Comment by smashwilson** [comment]
> Running that command in the terminal correctly produced the full list of branches.

Yah, this was fixed in atom/github#820, it just hasn't shipped yet. It'll be included in the next beta hotfix release :smile:

**Comment by r24y** [comment]
Oh, whoops! Thanks for letting me know, I forgot to search on closed issues 😅  keep up the excellent work!
